searchresults/index.html

```diff
@@ -44,40 +41,37 @@ <h1>Search Results</h1>
           <p id="confirmation"> ..................</p>
```

Comment:
the form on this page is supposed to "replicate the form on the first page" like my comment says. it does not do that. this form is completely different. I had it commented out because it wasn't what it was supposed to be. The form on the first page is already done and ready all you have to do is copy it basically. I made the form and it has all the correct things we need on it, and all you gotta do is just copy and paste. the form that you have has things like cities to select from, but we're not doing that yet, right now we're just going to have them write in their own city because we're trying to keep everything as basic as possible until the actual forms work. 
so, lines 14-45 need to be deleted and replaced with the form i made on page 1.